test/units/tests.ts

```diff
@@ -180,7 +181,8 @@ describe('VSOClient Units', function () {
 
         //Assert
         assert.equal(res.apiVersion, '1');
-        assert.equal(res.requestUrl, 'https://dev.azure.com/testTemplate?min=Wed%2C%2019%20Oct%200208%2000%3A00%3A00%20GMT');
+        var expectedURL = semver.lt(process.versions.node, '8.0.0') ? 'https://dev.azure.com/testTemplate?min=Wed%2C%2019%20Oct%20%20208%2000%3A00%3A00%20GMT' : 'https://dev.azure.com/testTemplate?min=Wed%2C%2019%20Oct%200208%2000%3A00%3A00%20GMT';
+        assert.equal(res.requestUrl, expectedURL);
```

Comment:
Could you describe these changes in more detail? Does it mean that returning value is different for Node 8?

Reply:
Yes, the returning value starts to differ from version 8 and upwards

Reply:
Could you please add short comment about it in code?

Reply:
Sure